client/ayon_substancepainter/plugins/publish/collect_textureset_images.py

```diff
@@ -187,13 +187,12 @@ def get_export_config(self, instance):
 
         creator_attrs = instance.data["creator_attributes"]
         preset_url = creator_attrs["exportPresetUrl"]
-        instance.data["anatomyData"] = instance.context.data["anatomyData"]
         self.log.debug(f"Exporting using preset: {preset_url}")
-
+        temp_dir = tempfile.TemporaryDirectory()
```

Comment:
I wonder if https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir this would be better intead - because `TemporaryDirectory` actually creates a directory, but we don't actually need to as far as I know because we'd never use the temp dir specified here to actually write to it?

What does Substance Painter do if we remove the "exportPath" here completely? or make it an empty string? :)

Reply:
> I wonder if https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir this would be better intead - because `TemporaryDirectory` actually creates a directory, but we don't actually need to as far as I know because we'd never use the temp dir specified here to actually write to it?
> 
> What does Substance Painter do if we remove the "exportPath" here completely? or make it an empty string? :)
Not working as the configuration needs it as mandatory.